atlasdb-config/src/test/java/com/palantir/atlasdb/memory/InMemoryAsyncAtlasDbFactoryTest.java

```diff
@@ -60,7 +60,7 @@ public void syncInitTimestampServiceSynchronous() {
     public void asyncInitTimestampServiceWithReadyKvsSynchronous() {
         KeyValueService kvs = createRawKeyValueService(false);
         TimestampService timestampService = factory.createManagedTimestampService(kvs, Optional.empty(), true);
-        assertThat(timestampService.isInitialized()).isTrue();
+        Awaitility.await().atMost(Duration.ofSeconds(2)).until(timestampService::isInitialized);
```

Comment:
Hmm, I'm not sure what this test is doing / if this assertion relates to the name of what the test claims to test any more. Looks like what the test wants is "if the KVS is initialised, then the timestamp service is also initialised", which won't be true here, so we have a semantic change.

This is for an in-memory test class, so I'd say if nothing breaks this semantic change is OK, though I think we should just remove this entire test - this is testing that it can be asynchronously initialised, which is tested by the test below already (i.e., `asyncInitTimestampServiceWithAsyncKvsAsynchronous()`).